docs/components/concepts/access-control/authorizations.md

```diff
@@ -0,0 +1,150 @@
+
+- Organization Admins
+- Organization Owners
+
+### Self-Managed Configuration
+
+Enable authorizations in your deployment using:
+
+```yaml
+camunda.security.authorizations.enabled: true
+```
+
+## Resources and Permissions
+
+### Available Resources
+
+The following table lists all resources that support authorization in Camunda 8 Orchestration Cluster, along with the available permissions per resource. This applies to Camunda 8 Orchestration Cluster (Zeebe, Operate, Tasklist, Orchestration Cluster APIs).
+
+| Resource Type                        | Resource Key Example                                    | Supported Permissions                                                                                                      |
+| ------------------------------------ | ------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------- |
+| **Application**                      | `Operate`, `Tasklist`, `Identity`, `*`                  | `ACCESS`                                                                                                                   |
+| **Authorization**                    | `*`                                                     | `CREATE`, `READ`, `UPDATE`, `DELETE`                                                                                       |
+| **Batch**                            | `*`                                                     | `CREATE`, `READ`, `DELETE`                                                                                                 |
+| **Decision Definition**              | `*`, `decisionId: order_decision`                       | `CREATE_DECISION_INSTANCE`, `READ_DECISION_DEFINITION`, `READ_DECISION_INSTANCE`, `DELETE_DECISION_INSTANCE`               |
+| **Decision Requirements Definition** | `*`, `decisionRequirementsDefinitionId: order_decision` | `CREATE`, `READ`, `UPDATE`                                                                                                 |
+| **Group**                            | `*`, `groupId: accounting`                              | `CREATE`, `READ`, `UPDATE`, `DELETE`                                                                                       |
+| **Mapping Rules**                    | `*`                                                     | `CREATE`, `READ`, `UPDATE`, `DELETE`                                                                                       |
+| **Message**                          | `*`                                                     | `CREATE`, `READ`                                                                                                           |
+| **Process Definition**               | `*`, `bpmnProcessId: order_process`                     | `CREATE_PROCESS_INSTANCE`, `READ_PROCESS_DEFINITION`, `READ_PROCESS_INSTANCE`, `READ_USER_TASK`, `UPDATE_PROCESS_INSTANCE` |
```

Comment:
🔧 I'm missing the `DELETE_PROCESS_INSTANCE` permission. It seems this isused by batch operations, although I wonder that's correct as a regular process instance cancellation requires the `UPDATE_PROCESS_INSTANCE` permission.

Reply:
These are two different things right? One is cancelling an instance and one is deleting it? 
So we should have DELETE_PROCESS_INSTANCE and UPDATE_PROCESS_INSTANCE? @remcowesterhoud

Reply:
What is deleting a process instance if not cancelling it? We don't have a delete process instance in Zeebe. We can delete the process definition, but that's covered in the resource authorization type.

Reply:
Sorry, I was already living in the future where we might have this deletion of instances for secondary storage: https://github.com/camunda/camunda/issues/30909#issue-2987797412
So I we will keep update process instance then only for now.